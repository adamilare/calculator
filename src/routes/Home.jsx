const Home = () => (
  <>
    <h1>Welcome to our page!</h1>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit. At facere unde
      ipsum quae quisquam sapiente provident harum repellat sit, temporibus
      corporis consequatur distinctio vero dolorum ab sunt tempora ipsa minus?
    </p>
  </>
);

export default Home;
