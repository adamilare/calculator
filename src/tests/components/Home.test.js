import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../routes/Home';

test('Home page displays correctly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
