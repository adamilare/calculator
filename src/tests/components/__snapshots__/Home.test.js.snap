// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page displays correctly 1`] = `
Array [
  <h1>
    Welcome to our page!
  </h1>,
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit. At facere unde ipsum quae quisquam sapiente provident harum repellat sit, temporibus corporis consequatur distinctio vero dolorum ab sunt tempora ipsa minus?
  </p>,
]
`;
